### ISSUE TITLE
about optimal permutation R

### ISSUE THREAD

**Issue by old-fan-kk** [facebookresearch/selavi#3]
Hi~,I didn't find the loss of optimal permutation R. Can you help me?

**Comment by duzexing** [comment]
It seems using gauss or zipf distribution to replace the uniform distribution. I don't know if i'm correct

**Comment by yudizhangzyd** [comment]
> It seems using gauss or zipf distribution to replace the uniform distribution. I don't know if i'm correct

Yeah, it looks like that, but why, isn't it inconsistent with the paper?

**Comment by hhc1997** [comment]
Does anyone know the meaning of [Rlogr]_y in Eq.(4). Thanks a lot.
